.github/workflows/api.yml

```diff
@@ -0,0 +1,37 @@
+name: api
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'api/**'
+      - .github/workflows/api.yml
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'api/**'
+      - .github/workflows/api.yml
+env:
+  PACKAGEPATH: ${{ github.workspace }}/go/src/github.com/${{ github.repository }}
+jobs:
+  protobuf-generate:
+    strategy:
+      matrix:
+        go-version: [1.14.x]
+        platform: [ubuntu-latest, macos-latest]
+    name: protobuf compile check
+    runs-on: ubuntu-latest
```

Comment:
while using the matrix, do we still have to use the `runs-on`?

Reply:
oops.

Reply:
[runs-on](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on) is required. This is the error in case we remove it: https://github.com/envoyproxy/xds-relay/pull/14/checks?check_run_id=488735330